Assets/MixedRealityToolkit/_Core/Interfaces/SpatialAwareness/Handlers/IMixedRealitySpatialAwarenessMeshHandler.cs

```diff
@@ -6,24 +6,27 @@
 
 namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.SpatialAwarenessSystem.Handlers
 {
-    public interface IMixedRealitySpatialAwarenessMeshHandler : IEventSystemHandler
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface IMixedRealitySpatialAwarenessMeshHandler<T> : IEventSystemHandler
```

Comment:
why change this interface to be a generic?

Reply:
Because it's more powerful and flexible as a generic. Just look how effective it is for the input system.

Then we can have different kinds of data types that can be returned using the handler.